tests/dapr_tests.mk

```diff
@@ -502,8 +503,11 @@ setup-disable-mtls:
 setup-app-configurations:
 	$(KUBECTL) apply -f ./tests/config/dapr_observability_test_config.yaml --namespace $(DAPR_TEST_NAMESPACE)
 
+setup-external-invocation-components: build-e2e-app-service_invocation_external push-e2e-app-service_invocation_external
```

Comment:
No need for a new target. Also, there is a separate step to push the app already.

Reply:
Fixed.